Add Hello in many more Indigenous languages.

erinishimoticha opened the pull request.

arecvlohe commented on src/components/Home.re:
```diff
@@ -255,6 +255,15 @@ let make = () => {
                 "Halito",
+                {j|ᎣᏏᏲ|j},
+                "Aaniin",
+                "Wáa sá i yatee",
```
Have to use `{j||j}` here as well for those special characters

arecvlohe commented on README.md:
```diff
@@ -39,7 +39,8 @@ Packages are managed using [yarn](https://yarnpkg.com/en/) for more deterministi
 - Clone this repository to your desktop (or somewhere else on your computer): `git clone git@github.com:nativesintech/natives-in-tech.git`
-- From the terminal, install dependencies: `$ yarn install`
+- From the terminal, install local dependencies: `$ yarn install`
+- From the terminal, install global dependencies: `npm i -g yarn phenomic npm-run-all`
```
Thanks!

arecvlohe replied:
Should an `.nvmrc` be added?

arecvlohe commented:
@all-contributors please add @erinishimoticha for code,doc

arecvlohe commented:
@all-contributors please add @erinishimoticha for code and doc

erinishimoticha commented:
Updated that string; added a missing global dependency to the README.

Test evidence: https://www.youtube.com/watch?v=s8OPC7KSAdI&feature=youtu.be

arecvlohe commented on src/components/Home.re:
```diff
@@ -250,11 +250,20 @@ let make = () => {
+                "Howa",               // Osage
+                "Ba'ax ka wa'alik",   // Mayan
+                "Niltse",             // Nahuatl
+                "Tanisi",             // Cree
```
👍
